VisualizationBase/src/items/ViewItem.cpp

```diff
@@ -36,6 +36,10 @@
 #include "RootItem.h"
 #include "renderer/ModelRenderer.h"
 
+#include "QJsonDocument"
```

Comment:
this is the wrong place to include QT headers. You should do this in `precompiled.h`. There are two options to do it there (read the comments in that file). Either include them such that all other projects will be able to use these headers (needed if you expose JSON stuff in the interface), or include them privately, so that only `VisualizatoinBase` knows about JSON. Use the second option if you only use JSON specific stuff in `.cpp` files. Either way, you'll need to move the includes to `precompiled.h`